packages/docs/src/examples/data-iterators/usage.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <v-container fluid grid-list-md>
+    <v-data-iterator
+      :items="items"
+      :items-per-page.sync="itemsPerPage"
+      :footer-props="{ itemsPerPageOptions }"
+    >
+      <template v-slot:default="props">
```

Comment:
@nekosaur Maybe use `#` shorthand for `v-slot`?

Reply:
We've decided to not use the shorthand in examples